prime-router/src/test/kotlin/fhirengine/azure/FHIRReceiverFilterIntegrationTests.kt

```diff
@@ -414,7 +414,7 @@ class FHIRReceiverFilterIntegrationTests : Logging {
             assertThat(event.params).isEqualTo(
                 mapOf(
                 ReportStreamEventProperties.FAILING_FILTERS to listOf("%resource.code.coding.code='1234'"),
-                ReportStreamEventProperties.FILTER_TYPE to ReportStreamFilterType.CONDITION_FILTER,
+                ReportStreamEventProperties.FILTER_TYPE to ReportStreamFilterType.CONDITION_FILTER.name,
```

Comment:
I think this was what Arnej was referring to in standup, but we should have integration tests for not filtering and filtering things out with the new routing filter setup. Looking over the integration scenarios we appear to have scenarios for everything but the regular routing filter so I guess this works out.